README.md

```diff
@@ -207,6 +207,54 @@ docker-compose run app yarn lint:diff
 docker-compose run app node_modules/.bin/eslint --fix path/to/file.js
 ```
 
+## Feature Flags
+Environment-specific feature flags are supported for both the api and frontend. Flagged features are assumed to be "off" unless the flag exists (and the value is truthy), thus feature flag conditions should always check for the presence or truthiness of the flag, not for it's absence. Environment variables for feature flags *MUST* be prefixed by `FEATURE_`, ex. `FEATURE_BRING_THE_AWESOME`.
```

Comment:
Suggestions for the example feature variable name [`FEATURE_BRING_THE_NOISE`](https://www.youtube.com/watch?v=kl1hgXfX5-U) or [`FEATURE_BRING_DA_RUCKUS`](https://www.youtube.com/watch?v=LRhRNgqPlyk) 😝